source/dotnet/wholesale-api/Events/Events.Infrastructure/InboxEvents/AggregatedTimeSeriesMessageFactory.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Azure.Messaging.ServiceBus;
+using Energinet.DataHub.Edi.Responses;
+using Energinet.DataHub.Wholesale.Events.Application.InboxEvents;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Energinet.DataHub.Wholesale.Events.Infrastructure.InboxEvents;
+
+public class AggregatedTimeSeriesMessageFactory : IAggregatedTimeSeriesMessageFactory
+{
+    /*
+     * THIS IS ALL MOCKED DATA
+     */
```

Comment:
How about making it available to intellisense?
```suggestion
    /// <summary>
    /// THIS IS ALL MOCKED DATA
    /// </summary>
```